Spigot-Server-Patches/0532-Add-PrepareSmithingEvent.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: William Blake Galbreath <Blake.Galbreath@GMail.com>
+Date: Fri, 3 Jul 2020 11:58:56 -0500
+Subject: [PATCH] Add PrepareSmithingEvent
+
+
+diff --git a/src/main/java/net/minecraft/server/ContainerSmithing.java b/src/main/java/net/minecraft/server/ContainerSmithing.java
+index 76a81b30fa9e53952202c61717231d68d7be8e71..5fb309e308433ac914f86d03e1459e490ab9ca3d 100644
+--- a/src/main/java/net/minecraft/server/ContainerSmithing.java
++++ b/src/main/java/net/minecraft/server/ContainerSmithing.java
+@@ -4,6 +4,7 @@ import java.util.List;
+ import javax.annotation.Nullable;
+ 
+ import org.bukkit.craftbukkit.inventory.CraftInventoryView; // CraftBukkit
++import org.bukkit.craftbukkit.inventory.CraftItemStack;
```

Comment:
Can't you move this to inline using the fqn? I still haven't figured out exactly when that should be done vs adding the import.

Reply:
Don't think it matters too much, just keep it consistent throughout?

Reply:
It should at least be paper commented; the import being there doesn't matter a whole lot even though inlining it would be easy.

Reply:
Commenting the imports isn't necessary, per aikar ;)